tests/test_correct_year.py

```diff
@@ -8,42 +8,30 @@
     :copyright: Copyright 2007-2016 by the Sphinx team, see AUTHORS.
```

Comment:
Review is **NOT** needed.
- Converting single test into pytest.mark.parametrize